resources/todomvc/architecture-examples/lit/package.json

```diff
@@ -10,7 +10,8 @@
         "dev": "wireit",
         "build": "wireit",
         "tsc": "wireit",
-        "rollup": "wireit"
+        "rollup": "wireit",
+        "copy-index-html": "wireit"
```

Comment:
I believe this npm script is unnecessary, as we'll probably never run it directly. It's good enough to have it in the `wireit` configuration below.